zulipterminal/cli/run.py

```diff
@@ -452,6 +452,30 @@ def list_themes() -> str:
     )
 
 
+def path_to_realm_zuliprc(realm_name_prefix: str) -> str:
```

Comment:
I'll wait to see this updated for the new version first, but I expect this and later commits would be the mapping of aliases to zuliprcs instead.